doc/source/tutorial/tutorial_presenting_larray_objects.ipyml

```diff
@@ -306,39 +296,63 @@ cells:
     A ``Session`` object is a dictionary-like object used to gather several arrays, axes and groups. 
     A session is particularly adapted to gather all input objects of a model or to gather the output arrays from different scenarios. Like with arrays, it is possible to associate metadata to sessions.
     
-    In addition to act like dictionaries, sessions offer several specific methods, like:
-    - ``save`` and ``load`` to save and load all arrays of a session at once in/from CSV or Excel or HDF5 file(s),
-    - ``equals`` and ``array_equals`` to compare arrays between two sessions (scenarios) one by one,
-    - ``apply`` to apply a function to all arrays of a session.
+    See the [Session](../api.rst#session) section of the API Reference to explore all methods of Session objects.
```

Comment:
Links to API reference should be **after** presenting each kind of object. and the other links (to load/dump) and general working should come after this section too I think.